### PR TITLE
updated api urls

### PR DESCRIPTION
**Pull request by roy-sagar11**

### CODE REVIEW

File: app/src/hooks/projectshook.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const useProjectsHook = () => {
   const getProjectById = async (projectId: string) => {
     try {
       setFetching(true);
-      const response: Project = await AxiosPublic.get(`/project/${projectId}`);
+      const response: Project = await AxiosPublic.get(`/api/v1/project/${projectId}`);
```
Comment (will-freshworks) [inline app/src/hooks/projectshook.ts:15]: This should really be parameterised somewhere. Can you add a base URL to AxiosPublic or a constant somewhere?

**Review by will-freshworks — CHANGES_REQUESTED** [review]

File: app/src/hooks/projectshook.ts (hunk 1 above)
Comment (matbusby-fw) [reply app/src/hooks/projectshook.ts:15]: This had been set in the axios config through an env variable previously.  Not sure if that changed.

Comment (matbusby-fw) [reply app/src/hooks/projectshook.ts:15]: `REACT_APP_BASE_API_URL=http://localhost:4000/api/v1`

That was used in app/src/hooks/axios.ts to set the BaseURL for Axios.  You may have just been missing that env.

Comment (will-freshworks) [reply app/src/hooks/projectshook.ts:15]: Future task will be to make these consistent and set in a single place. REACT_APP_BASE_API_URL makes sense as a single place to set this.

**Pull request merged**